### ISSUE TITLE
Twine Inaccessible For Users Of Screen Readers

### ISSUE THREAD

**Issue by HiEv** [klembot/twinejs#607]
**Describe the bug**
I'm re-opening "[Twine Inaccessible For Users Of Screen Readers](https://github.com/klembot/twinejs/issues/564)" issue #564.

The Twine application interface needs to be made accessible to those with visual and/or physical impairments which make it difficult for them to navigate using screen readers and the keyboard.  For keyboard navigation problems see also "[Cannot open passage editor using keyboard alone](https://github.com/klembot/twinejs/issues/565)" issue #565.

There are multiple issues with the current Twine interface, described below:

**To Reproduce**
Steps to reproduce the issues in the main Twine window:
1. Install a screen reader, like [NVDA](https://www.nvaccess.org/download/).
2. In the main Twine window, hover your mouse over the story image or the story configuration icon.
3. Hear "button" on all of them.
4. Try to navigate the main Twine window using the TAB key.
5. No visual indicator of an item having focus in most cases, and the same problem with most items only being labeled "button".

Steps to reproduce the issues in a Twine story window:
1. Install a screen reader, like [NVDA](https://www.nvaccess.org/download/).
2. In the story window, try to navigate to the story nodes using the TAB key, you can't do it.
3. No visual indicator of an item TABbed to having focus in most cases, though the items are described properly by the screen reader.
4. No easy way to find story nodes, as the node name is only given if you hover your mouse over that specific bit of text within a node box.
5. The background behind the story nodes is also unhelpfully labeled as "graphic".

**Expected behavior**
All items should be clearly labeled.  The story images should probably either use an ["`aria-labelledby`" attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) to refer to the text link to the corresponding story (and yes, it's spelled "labelled"), or an ["`aria-label`" attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) if there isn't a corresponding label element.  The description of the story images should say something like, "Story name. 1 of 10", if there are 10 stories (punctuation is needed to add correct pauses).

The config icons definitely need an "`aria-label`" attribute.  The description of the config icons should say something like, "Configure, Story name."

Story nodes need a method of navigation via. keyboard.  When the mouse hovers over a story node, it should always say the name of the story node, instead of just whatever is directly under the mouse.  The background shouldn't be labeled.

** App version **
Tested with the desktop app version of Twine v2.3.4.

** Environment **
Tested on Windows v7sp1.

**Comment by klembot** [comment]
I've edited the title of this bug to be a bit more specific. Thanks for the details here.
